packages/console/src/components/common/MapTaskExecutionsList/TaskNameList.tsx

```diff
@@ -68,7 +72,7 @@ export const TaskNameList = ({
               color={log.uri ? 'primary' : 'textPrimary'}
               onClick={log.uri ? handleClick : undefined}
               key={taskLogName}
```

Comment:
can you add this key to the div that's wrapping this please?
 <div
            style={{
              display: 'flex',
              alignItems: 'center',
            }}
            key={taskLogName}
          >